app/models/package.scala

```
@@ -19,6 +19,7 @@ import play.api.libs.json._
 import scala.annotation.nowarn
 
 package object models {
+  type Result[T] = Either[Throwable, T]
 
```

Saves having to write long form out everywhere we can return an error in an Either.